apps/ledger-live-desktop/src/newArch/features/Collectibles/Ordinals/components/Inscriptions/DetailsDrawer/index.tsx

```diff
@@ -1,18 +1,66 @@
 import React from "react";
-import { SideDrawer } from "~/renderer/components/SideDrawer";
 import { SimpleHashNft } from "@ledgerhq/live-nft/api/types";
+import { DetailDrawer } from "LLD/features/Collectibles/components";
+import { CollectibleTypeEnum } from "LLD/features/Collectibles/types/enum/Collectibles";
+import { Flex, Icons, Text } from "@ledgerhq/react-ui";
+import Button from "~/renderer/components/Button";
+import useInscriptionDetailDrawer from "./useInscriptionDetailDrawer";
+import { t } from "i18next";
+
+type ViewProps = ReturnType<typeof useInscriptionDetailDrawer> & {
+  onClose: () => void;
+};
 
 type Props = {
   inscription: SimpleHashNft;
   onClose: () => void;
+  isLoading: boolean;
 };
-const InscriptionDetailsDrawer: React.FC<Props> = ({ inscription, onClose }) => {
-  // will be replaced by DetailsDrawer from collectibles
-  return (
-    <SideDrawer direction={"left"} isOpen={!!inscription} onRequestClose={onClose}>
-      {inscription.name || inscription.contract.name}
-    </SideDrawer>
-  );
+
+const View: React.FC<ViewProps> = ({ data, onClose }) => (
+  <DetailDrawer
+    collectibleType={CollectibleTypeEnum.NFT}
+    areFieldsLoading={data.areFieldsLoading}
+    collectibleName={data.collectibleName}
+    contentType={data.contentType}
+    collectionName={data.collectionName}
+    details={data.details}
+    previewUri={data.previewUri}
+    originalUri={data.originalUri}
+    isPanAndZoomOpen={data.isPanAndZoomOpen}
+    mediaType={data.mediaType}
+    tags={data.tags}
+    useFallback={data.useFallback}
+    tokenId={data.tokenId}
+    isOpened={true}
+    closeCollectiblesPanAndZoom={data.closeCollectiblesPanAndZoom}
+    handleRequestClose={onClose}
+    openCollectiblesPanAndZoom={data.openCollectiblesPanAndZoom}
+    setUseFallback={data.setUseFallback}
+  >
+    <DetailDrawer.Actions>
+      <Flex>
+        <Button
+          outlineGrey
+          style={{ flex: 1, justifyContent: "center" }}
+          my={4}
+          onClick={() => console.log("hide " + data.collectibleName + " clicked")}
```

Comment:
if there is no significant handler here, delete this console.log and make a TODO comment